modules/abstract-utxo/src/transaction.ts

```diff
@@ -206,12 +206,26 @@ export function explainPsbt<TNumber extends number | bigint>(
   } catch (e) {
     throw new Error('failed to parse psbt hex');
   }
+  const txOutputs = psbt.txOutputs;
+  function getChangeAddresses() {
+    try {
+      return utxolib.bitgo
+        .findInternalOutputIndices(psbt)
+        .map((i) => utxolib.address.fromOutputScript(txOutputs[i].script, network));
+    } catch (e) {
+      if (e instanceof Error && e.message === 'No multi sig input found') {
```

Comment:
instead of relying on error message we should have a dedicated error class

```
class ErrorNoMultiSigInputFound extends Error { ... }

...
if (e instanceof ErrorNoMultiSigInputFound) { ... }
```

but why not propagate the error instead?

Reply:
> why not propagate the error instead?
As per existing test cases, `explainTransaction` should return empty change outputs for a PSBT with no multi-sig inputs (ex: PSBT only with `p2shP2pk`). It makes sense because it's not in `explainTransaction`'s scope to assert for presence of mult-sig inputs in PSBTs.